modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```
@@ -877,37 +877,40 @@ Resources:
                   local_UseCfnUrl: !If [UseCfnUrl, !Sub '--url ${CfnEndpointUrl}', '']
             'c:\cfn\scripts\watchmaker-install.ps1':
               content:
-                !If
-                  - UseWamStandalone
-                  - !Sub |
-                      # Download watchmaker standalone pkg
+                !Join
+                  - ''
+                  - - !Sub |
                       [Net.ServicePointManager]::SecurityProtocol = "Ssl3, Tls, Tls11, Tls12"
```

I think this is a scenario where it is cheaper to duplicate the one line, instead of the extra nest/indent... :) Though, we also ought to be able to restrict the protocols now...

```suggestion
                      [Net.ServicePointManager]::SecurityProtocol = "Tls12, Tls13"
```

---

Lol!  Yeah. I think I _might_ have something but if not, I'll just duplicate.

---

Okay.  Cheaper wins.  I'm just going to duplicate the line.